CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 * [Viz Builder] Fixes time series for new chart types ([#2309](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2309))
 * [Viz Builder] Add index pattern info when loading embeddable ([#2363](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2363))
 * Fixes management app breadcrumb error ([#2344](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2344))
+* Fix leaflet-vega and vega-interpreter mismatch in yarn.lock file ([#2432](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2432))
```

Comment:
This is a good new habit, although I'm not sure a change like this warrants an update in the changelog. But let's stick with it for now.

Reply:
Yep. Every PR does not need to have a corresponding entry in the changelog file.

Reply:
removed it from the CHANGELOG.md file